File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    y = x.toString()
    for(let i = 0; i < y.length / 2; i++){
        if(y[i] !== y[y.length - (i+1)]){
            return false
        }
        
    }
    return true
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        y = str(x)
        i = 0
        j = len(y) // 2
        while i < j:
            if y[i] != y[len(y) - (i+1)]:
                return False
            i = i + 1
        return True